File: pekko-sample-cluster-docker-compose-java/src/main/java/com/example/ClusteringApp.java
package com.example;

import akka.actor.typed.ActorSystem;
import com.typesafe.config.Config;
import com.typesafe.config.ConfigFactory;

public class ClusteringApp {

  public static void main(String[] args) {
    Config config = ConfigFactory.load();
    String clusterName = config.getString("clustering.cluster.name");

    ActorSystem<?> system = ActorSystem.create(ClusterListener.create(), clusterName);
  }

}

File: pekko-sample-cluster-docker-compose-scala/src/main/scala/com/example/ClusteringApp.scala
package com.example

import org.apache.pekko.actor.typed.ActorSystem
import com.typesafe.config.ConfigFactory

object ClusteringApp {
  def main(args: Array[String]): Unit = {
    val config = ConfigFactory.load()
    val clusterName = config.getString("clustering.cluster.name")

    ActorSystem(ClusterListener(), clusterName)
  }
}
